PersonenOrt.Repository.Service/Controllers/PersonController.cs

```diff
@@ -25,36 +25,51 @@ public IEnumerable<Person> Get()
             }
         }
         [HttpPut("{id:int}")]
-        public Person PutPerson(int id, Person person)
+        public String PutPerson(int id, Person person)
```

Comment:
Hier passt Pfad-Paremeter und funktoins-signatur zusammen -> du kannst den parameter im Pfad (.../pfadparameter) in der Funktion verwenden...